src/ProtectedRoute.tsx

```diff
@@ -5,24 +5,37 @@ import ConditionalWrapper from 'components/utils/ConditionalWrapper';
 import { STATIC_ROUTES } from 'utils/routes';
 import { useUser } from 'store/user';
 import { REDIRECT_URI_KEY } from 'common/constants';
+import { usePersona } from 'store/persona';
```

Comment:
if easy, create another page only for accepted terms but otherwise forget about it for now